src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/import.py

```diff
@@ -0,0 +1,90 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+from azure.mgmt.containerregistry.v2018_02_01_preview.models import (
+    ImportImageParameters,
+    ImportSource,
+    ImportMode
+)
+from ._utils import (
+    validate_managed_registry,
+    get_registry_by_login_server,
+    get_registry_by_registry_name
+)
+
+SOURCE_REGISTRY_MISING = "Please specify the source container registry:"
+IMPORT_NOT_SUPPORTED = "Image imports are only supported for managed registries."
```

Comment:
NIT: not sure if you want to mention `image` any more.

Reply:
Deleted